app/views/ops/_diagnostics_database_tab.html.haml

```diff
@@ -1,38 +1,6 @@
-- @angular_form = true
-
-%form#form_div.form-horizontal{:name                                  => "angularForm",
-                              'ng-controller'                         => "diagnosticsDatabaseFormController as vm",
-                              'ng-show'                               => "vm.afterGet",
-                              'data-db-backup-form-field-changed-url' => "/#{controller_name}/db_backup_form_field_changed/",
-                              'data-submit-url'                       => "/#{controller_name}/db_backup/",
-                              'miq-form'                              => true,
-                              'model'                                 => 'vm.diagnosticsDatabaseModel',
-                              'model-copy'                            => 'vm.modelCopy',
-                              'form-changed'                          => true,}
-  - if @sb[:active_tab] == "diagnostics_database"
-    %h3
-      = _("Basic Info")
-  .form-group
-    %label.col-md-2.control-label
-      = _("Type")
-    .col-md-8
-      %p.form-control-static
-        = h(@database_display_name)
-  .form-group
-    %label.col-md-2.control-label
-      = _("Hostname")
-    .col-md-8
-      %p.form-control-static
-        = h(@database_details["host"])
-  .form-group
-    %label.col-md-2.control-label
-      = _("Database Name")
-    .col-md-8
-      %p.form-control-static
-        = h(@database_details["database"])
-  .form-group
-    %label.col-md-2.control-label
-      = _("Username")
-    .col-md-8
-      %p.form-control-static
-        = h(@database_details["username"])
+- if @sb[:active_tab] == "diagnostics_database"
+  = react('MiqStructuredList', {:title   => @data[:title],
+                            :headers => @data[:headers],
+                            :rows    => @data[:rows],
+                            :message => @data[:message],
+                            :mode    => "miq_summary #{@data[:mode]}"})
```

Comment:
it can look something like this-
`<%= diagnostic_database_list(@data) if @sb[:active_tab] == "diagnostics_database" %>`